build/npm/import-language-xlfs.js

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+let path = require("path");
+let gulp = require('gulp');
+let minimist = require('minimist');
+
+function update(options) {
+	let location = options._;
+
+	gulp.src(`${location}/drop/loc/**/*.xlf`)
+		.pipe(gulp.dest(function (file) {
+			let firstSlashIndex =  file.relative.indexOf('\\');
+			let language = '\\' + file.relative.substr(0,firstSlashIndex) + '\\';
+			file.path = file.base + language + path.basename(file.relative);
+			return './resources/xlf/'
+		}));
+}
+
+if (path.basename(process.argv[1]) === 'import-language-xlfs.js') {
```

Comment:
Any particular reason you made this a standalone script like this instead of a gulp task like you have for import-extensions-xlfs? Doesn't matter too much I guess but it's usually nice to keep similar pieces of code near each other.

Reply:
Arguments for gulp task don't seem to exist from what I've been reading (this requires downloading extracting the drop folder), also the internal folder structure of the drop file is different from the actual structure of the resources/xlf folder. (I.E. In the drop folder, it is <language_id>/resources/xlf/en" instead of "resources/xlf/<language_id>"

Reply:
What do you mean "requires downloading extracting the drop folder"? And how does that have anything to do with this being a script?

Is it expected that the user has to pass in a folder location that may be outside the ADS folder? That definitely wasn't clear so if that's the case then I guess this is fine but you definitely should be documenting that. 

Where is this drop folder coming from and is there any way you could have the contents of this folder be created in the folder structure that we actually want?

Reply:
You have to download this drop zip file containing the  XLFs from https://dev.azure.com/ms/azuredatastudio/_build?definitionId=453 and extracting the folder. I mentioned in my documentation that you needed to download a drop file from this pipeline, to do this. I'm not sure why the contents of the drop folder are created this way, but it appears to be related to something in LocProject.json. I'm going to investigate why this is the case.

Reply:
Ok, while you're investigating that could you remove this script from this PR for the time being? The rest of this looks fine so we can get that merged in - and then follow up with this later on once we figure out what the best way to approach that is.

Reply:
Sure thing.